index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+    <title>Adopt A Dog</title>
+</head>
+<body>
+  <div class="grid-container">
```

Comment:
instead of using a div for this, there might be a HTML5 element you can use instead.